CodeReview/src/CodeReviewManager.cpp

```diff
@@ -81,7 +85,8 @@ void CodeReviewManager::saveReview(QString newVersion)
 
 }
 
-NodeReviewsList* CodeReviewManager::loadReview(QString newVersion)
+NodeReviewsList* CodeReviewManager::loadReview(QString newVersion, VersionControlUI::DiffSetup& diffSetup,
+																 Visualization::ViewItem* viewItem)
```

Comment:
So, looking at the `load()` call below, I see that `newVersion` contains both the version and the name. Could we separate these two? Here you would need some way of fetching the manager name and prepending it. Perhaps that can be carried inside the diffSetup. It just somehow feels like a bad idea that the `revsionName` of a manager also contains the manager name.